deploy/import-upstream.sh

```diff
@@ -1,2 +1,14 @@
-oc import-image coreos/prometheus-operator:v0.25.0 --from=quay.io/coreos/prometheus-operator:v0.25.0 --confirm
-oc import-image coreos/configmap-reload:v0.0.1 --from=quay.io/coreos/configmap-reload:v0.0.1 --confirm
+#!/bin/sh
+oc import-image prometheus:v2.11.0 --from=quay.io/openshift/origin-prometheus:v3.11 --confirm
+oc import-image prometheus-operator:latest --from=quay.io/openshift/origin-prometheus-operator:v3.11 --confirm
+oc import-image prometheus-configmap-reloader:latest --from=quay.io/openshift/origin-configmap-reload:v3.11 --confirm
+oc import-image prometheus-config-reloader:latest --from=quay.io/openshift/origin-prometheus-config-reloader:v3.11 --confirm
+oc import-image prometheus-alertmanager:v0.15.0 --from=quay.io/openshift/origin-prometheus-alertmanager:v3.11 --confirm
+
+oc import-image amq-interconnect:1.4-6 --from=quay.io/interconnectedcloud/qdrouterd:1.7.0 --confirm
+oc import-image amq-interconnect-operator:latest --from=quay.io/interconnectedcloud/qdr-operator:1.0.0-beta2 --confirm
+
+oc import-image smart-gateway --from=quay.io/redhat-service-assurance/smart-gateway:latest --confirm
```

Comment:
These two might as well get `:latest` on the left hand side for consistency.

Not sure we want to use `:latest` on the right hand side, but still okay with approving it for now.

Reply:
Done